src/Service/WebconfigProvider.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+* Configuration Service Provider for Joomla Webservices.
+*
+* @package    Webservices
+* @copyright  Copyright (C) 2004 - 2016 Open Source Matters, Inc. All rights reserved.
+* @license    GNU General Public License version 2 or later; see LICENSE.txt
+*/
+
+namespace Joomla\Webservices\Service;
+
+use Joomla\DI\Container;
+use Joomla\DI\ServiceProviderInterface;
+use Joomla\Registry\Registry;
+
+/**
+* Configuration service provider
+*
+* @since  1.0
+*/
+class WebconfigProvider implements ServiceProviderInterface
+{
+	/**
+	* Configuration instance
+	*
+	* @var    Registry
+	* @since  1.0
+	*/
+	private $config;
+
+	/**
+	* Constructor.
+	*
+	* @since   1.0
+	* @throws  \RuntimeException
+	*/
+	public function __construct()
+	{
+		// Set the configuration file path for the application.
+		$file = JPATH_API . '/config.dist.json';
```

Comment:
I only need the structure of the JSON, not the credentials because i get the credentials from the current database instance.